packages/kit/src/core/sync/create_manifest_data/index.js

```diff
@@ -192,7 +192,7 @@ function create_routes_and_nodes(cwd, config, fallback) {
 			// process files first
 			for (const file of files) {
 				if (file.is_dir) continue;
-				if (!file.name.startsWith('+')) continue;
+				if (!file.name.startsWith('+') || file.name.endsWith('.d.ts')) continue;
```

Comment:
mhhhhm or should we scope it to "file ends with `.d.ts` but has to have one of the known file names"? So that you can't do `+yolo.d.ts`

Reply:
blurgh i guess we probably should